.github/CODEOWNERS

```diff
@@ -1,5 +1 @@
-# See https://help.github.com/articles/about-codeowners/ for more info
-# Each line is a file pattern followed by one or more owners.
-
-# These owners will be the default owners for everything in the repo.
-* @seek-oss/logger-js-maintainers
+* @seek-oss/sig-backend-tooling
```

Comment:
Not sure if we need the renovate things as well? Similar to [here](https://github.com/seek-oss/skuba/blob/286024340221aa9d3424d717ec8b14bb32c93e95/.github/CODEOWNERS).

Reply:
Ah, could do. That would let us auto-merge lock file maintenance and the like.